src/java/org/apache/cassandra/utils/concurrent/LoadingMap.java

```diff
@@ -52,14 +56,24 @@
     // It also ensures exactly-once semantics for the update operation.
     private final Map<K, CompletableFuture<V>> internalMap;
 
+    private final Duration timeout;
+
     public LoadingMap()
     {
-        this.internalMap = new NonBlockingHashMap<>();
+        this(8);
```

Comment:
8 is the actual default for NonBlockingHashMap

Reply:
I'd rather we used an artificial value here, and then handled it in the constructor, like:
```
if (initialSize == DEFAULT) 
{
   internalMap = new NonBlockingHashMap<>();
}
else
{
    internalMap = new NonBlockingHashMap<>(initialSize);
}
```

or at least make a constant named `DEFAULT_NON_BLOCKING_HASH_MAP_SIZE` or sth along this line